### ISSUE TITLE
Small aesthetic bug in the Whitelist_CDN function

### ISSUE THREAD

**Issue by wbartels** [Adamm00/IPSet_ASUS#22]
###### Brief Description Of Issue
The Whitelist_CDN function introduces an extra whitespace in the skynet.ipset
```
create Skynet-Whitelist hash:net family inet hashsize 2048 maxelem 65536 comment
add Skynet-Whitelist 23.10.80.0/20 comment "CDN-Whitelist: AS16625 "
add Skynet-Whitelist 23.212.16.0/20 comment "CDN-Whitelist: AS16625 "
add Skynet-Whitelist 52.119.249.0/24 comment "CDN-Whitelist: Amazon "
add Skynet-Whitelist 13.48.0.0/15 comment "CDN-Whitelist: Amazon "
add Skynet-Whitelist 184.84.192.0/22 comment "CDN-Whitelist: AS16625 "
add Skynet-Whitelist 35.158.127.64/26 comment "CDN-Whitelist: Amazon "
add Skynet-Whitelist 35.180.112.80/29 comment "CDN-Whitelist: Amazon "
add Skynet-Whitelist 151.139.88.0/23 comment "CDN-Whitelist: AS33438 "
add Skynet-Whitelist 17.132.13.0/24 comment "CDN-Whitelist: AS714 "
add Skynet-Whitelist 23.197.48.0/23 comment "CDN-Whitelist: AS16625 "
```

###### Fix
Remove the space in the firewall file just before `\"\n"` on line 817, 820 and 821

**Comment by Adamm00** [comment]
Fixed in [9de61c6](https://github.com/Adamm00/IPSet_ASUS/commit/9de61c6140ca4abeaabe57548b7666cdcfca4d8f)

**Comment by wbartels** [comment]
Thanks!
